MS5611.h

```diff
@@ -19,6 +19,14 @@
 #define MS5611_ERROR_2                        2         // low level I2C error
 #define MS5611_NOT_READ                       -999
 
+enum osr_t
+{
+    OSR_ULTRA_HIGH = 12, // 10 millis
```

Comment:
timing measured on an UNO?

Reply:
Timings were not measured, they are the values that correspond to the del[] array in the convert function.